site/content/en/docs/manual/advanced/analytics-and-monitoring/auto-qa.md

```diff
@@ -78,90 +164,179 @@ To create a **Ground truth** job, do the following:
+However, it's not possible to select new validation frames after the task is created.
 
-The Analytics page has the following fields:
+Parameters:
+- frame count per job (%) - the percent of job frames (segment size) to be **added** into each
+annotation job from the validation set
+- total frame count (%) - the percent of the task frames to be included into the validation set.
+This value must result in at least `frame count per job` * `segment size` frames.
 
-<!--lint disable maximum-line-length-->
+### Mode summary
 
-| Field                   | Description                                                                                                                                                                                       |
-| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Mean annotation quality | Displays the average quality of annotations, which includes: the count of accurate annotations, total task annotations, ground truth annotations, accuracy rate, precision rate, and recall rate. |
-| GT Conflicts            | Conflicts identified during quality assessment, including extra or missing annotations. Mouse over the **?** icon for a detailed conflict report on your dataset.                                 |
-| Issues                  | Number of {{< ilink "/docs/manual/advanced/analytics-and-monitoring/manual-qa" "opened issues" >}}. If no issues were reported, will show 0.                                                                                                |
-| Quality report          | Quality report in JSON format.                                                                                                                                                                    |
-| Ground truth job data   | "Information about ground truth job, including date, time, and number of issues.                                                                                                                  |
-| List of jobs            | List of all the jobs in the task                                                                                                                                                                  |
+Here is a brief summary of the validation modes:
 
-<!--lint enable maximum-line-length-->
+| **Aspect**         | **Ground Truth**                                 | **Honeypots**                         |
+| -------------- | -------------------------------------------- | ------------------------------------------- |
+| When can be used        | any time                          | at task creation only |
+| Frame management options       | add, remove, exclude, restore     | exclude, restore, change honeypots in jobs |
+| Task frame requirements        | -                          | random ordering only |
```

Comment:
video data is not supported with the `Honeypots` validation mode

Reply:
Is it possible to create a task with random ordering from a video?

Reply:
I meant that probably it also should be mentioned in the summary table